tests/console/yast2_nfs_server.pm

```diff
@@ -26,20 +26,24 @@ use mmapi;
 use mm_network;
 
 sub run {
+    my ($self) = @_;
     select_console 'root-console';
 
     if (get_var('NFSSERVER')) {
         # Configure static IP for client/server test
         configure_default_gateway;
-        configure_static_ip('10.0.2.1/24');
+        configure_static_ip('10.0.2.101/24');
         configure_static_dns(get_host_resolv_conf());
+
+        systemctl 'stop ' . $self->firewall;
+        systemctl 'disable ' . $self->firewall;
```

Comment:
Doesn't look right to me - YaST should offer the option to configure the firewall for this.

Reply:
@Vogtinator I can do that change on SLE12-SP3 where this [option](http://pdostal-server.suse.cz/tests/1736#step/yast2_nfs_server/37) is available. Unfortunately the same [option](http://pdostal-server.suse.cz/tests/1742#step/yast2_nfs_server/37) is unavailable in SLE15.

Reply:
That's unfortunate... So this workaround should be fine.

Reply:
Here it is: [qam-nfs-server@sles15](http://pdostal-server.suse.cz/tests/1771) [qam-nfs-server@sles12-sp3](http://pdostal-server.suse.cz/tests/1773#) [qam-nfs-server@sles12-sp4](http://pdostal-server.suse.cz/tests/1775). Do you agree with that @Vogtinator ?

Reply:
I would be fine with the workaround already - but for testing the firewall rule properly I'd recommend an assert_screen after the "mysterious keypresses".